scripts/update-bcd.ts

```diff
@@ -679,6 +684,48 @@ const skipCurrentBeforeSupport = skip("currentBeforeSupport", ({
   }
 });
 
+export const hasSupportMatrixContradictions = (
```

Comment:
I suggested the "contradictions" terminology, but it's not always a case of a contradiction, and can be a refinement of support. For example, we can sometimes update versions using `≤` based on test results, but it's not a contradiction, just more information.

So I'd like to sugget this terminology instead:

```suggestion
export const hasSupportUpdates = (
```

Reply:
implemented in https://github.com/openwebdocs/mdn-bcd-collector/pull/909/commits/f27f002479e1c00843d4faee6b9ce72aeb1a6de0